pkgs/top-level/aliases.nix

```
@@ -727,6 +727,7 @@ mapAliases {
   ''; # Added 2024-10-12
   git-backup = throw "git-backup has been removed, as it has been abandoned upstream. Consider using git-backup-go instead.";
   git-credential-1password = throw "'git-credential-1password' has been removed, as the upstream project is deleted."; # Added 2024-05-20
+  git-stree = throw "'git-stree' has been deprecated by upstream. Consider using the recommended replacement 'git-subrepo' instead"; # Added 2025-05-05
```

> the recommended replacement

this is a little ambiguous. Recommended by who? Either:
```suggestion
  git-stree = throw "'git-stree' has been deprecated by upstream. Consider using 'git-subrepo' instead"; # Added 2025-05-05
```

Or:
```suggestion
  git-stree = throw "'git-stree' has been deprecated by upstream. Upstream recommends using the replacement 'git-subrepo' instead"; # Added 2025-05-05
```